ps_emailsubscription.php

```diff
@@ -395,6 +395,21 @@ public function newsletterRegistration($hookName = null)
             }
         }
 
+        // hook for newsletter registration/unregistration : fill-in hookError string is there is an error
+        $hookError = null;
+        Hook::exec(
+            'actionNewsletterRegistration', 
```

Comment:
```suggestion
            'actionNewsletterRegistrationBefore', 
```

And could be cool to have the after, and while register / unregister, wdtyt?